<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTtestClass</class>
 <widget class="QMainWindow" name="QTtestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Othello Stone!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>싱글플레이</string>
    </property>
    <widget class="QPushButton" name="Singleplay_man">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>영혼의 맞다이</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Singleplay_com">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>AI 대난투</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>멀티플레이</string>
    </property>
    <widget class="QPushButton" name="Multiplay_man">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>직접 멀티플레이</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="svr_ip">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:7.875pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;113.30.24.43&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="svr_port">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:7.875pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;8472&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Multiplay_com">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>신속정확 대리 플레이</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTtest.qrc"/>
 </resources>
 <connections/>
</ui>
